src/selectors/entities/files.ts

```diff
@@ -1,20 +1,19 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import * as reselect from 'reselect';
+import {createSelector} from 'reselect';
 
 import {getCurrentUserLocale} from 'selectors/entities/i18n';
 
-import {sortFileInfos} from 'utils/file_utils';
-
-import * as types from 'types';
 import {GlobalState} from 'types/store';
 
-function getAllFiles(state: types.store.GlobalState) {
+import {sortFileInfos} from 'utils/file_utils';
+
+function getAllFiles(state: GlobalState) {
     return state.entities.files.files;
 }
 
-function getFilesIdsForPost(state: types.store.GlobalState, postId: string) {
```

Comment:
this `types.whatever.Type` references here, aren't an accident, that was introduced to fix a problem to use it from the mattermost-webapp. If you remove them, maybe you break the mattermost-webapp build, or... maybe is already fixed and this is no longer needed.

Reply:
I made a quick try and is working locally.